doc/api/ffi.md

```diff
@@ -0,0 +1,132 @@
+# Foreign Function Interface (FFI)
```

Comment:
Thanks for this incredible work! 👏🏼 

There are two things I'd like to see in this documentation:
- More code examples (at least one general example at the beginning) 
- A little educational paragraph to explain the pros and cons of using ffi instead of napi.

Thanks again ^^